pulsar-client-api/src/main/java/org/apache/pulsar/client/api/CursorClient.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.api;
+
+import java.io.Closeable;
+import java.util.concurrent.CompletableFuture;
+import org.apache.pulsar.common.classification.InterfaceAudience;
+import org.apache.pulsar.common.classification.InterfaceStability;
+
+/**
+ * A {@link CursorClient} connects to a PersistentTopic and can manage cursor data of a persistent subscription.
+ * This is an interface that abstracts behavior of Pulsar's cursor client.
+ * This {@link CursorClient} is used by a readonly topic owner.
+ *
+ * @since 2.9.0
+ */
+@InterfaceAudience.Private
+@InterfaceStability.Evolving
+public interface CursorClient extends Closeable {
```

Comment:
this is a new API and it is not cited in PIP-63 
https://github.com/apache/pulsar/wiki/PIP-63%3A-Readonly-Topic-Ownership-Support 

can you please update the document or start a new one ?

Reply:
@eolivelli Yes, it is a little different regarding to the complexity of implementation, but the main idea is the same.
And ...
This PIP doc is in wiki directory, actually I don't know how to update that. Any help?

Reply:
@eolivelli Hi, I updated the doc in this PR doc. Please take a look.